code/modules/events/immovable_rod.dm

```diff
@@ -27,7 +27,7 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 	announceWhen = 5
 
 /datum/round_event/immovable_rod/announce(fake)
-	priority_announce("What the fuck was that?!", "General Alert")
+	priority_announce("Unknown high-density object detected on collision co-- What the fuck was that?!", "General Alert")

```

Comment:
I want to play with the formatting for this a little stille, probably italicizing the "what the fuck" part of it for emphasize, though I don't yet know how to make that work correctly

Any other suggestions for wording, are gladly welcome still.